package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "stytch-b2b-nextjs-example",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 3,
```

Comment:
should we put these dependency changes in a separate PR?

Reply:
We can't right? OIDC depends on version `6.2.0` of `stytch-node`.